source/includes/_bulk_data.md.erb

```diff
@@ -20,6 +20,15 @@ The bulk data webhooks include exports of the following tables:
 
 For full information on the schema of each table, use the `/api/bulk_data/schema.json` API endpoint.
 
+## Bulk Data Files
+
+Each table exposed by the bulk data API is made available as a CSV file with the URL to download each file sent via webhook.
+
+We expire access to data 24 hours after it has been generated. This means that if you are building an automated system
+to ingest data from this API it must process webhook notifications within 24 hours.
+
+Once webhook data has expired, the download URLs will return a 403 error response.
+
 ### Interpreting the share_clicks table
```

Comment:
Total nitpick: This level 3 heading was under the level 2 heading "Bulk Data Data Schemas", but now it's under "Bulk Data Files". Is this intentional?